src/router.ts

```diff
@@ -111,18 +111,13 @@ const routes:RouteRecordRaw[] = [
       {
         component: MailingListCreate,
         name: 'Create Mailing List',
-        path: '/mailing_list/create'
+        path: '/mailing_list/create/:canvasSiteId?'
```

Comment:
@pauline2k - did you intend the path to end with "?"

Reply:
@johncrossman Yes, the "?" makes it optional. This seems to be the way vue-router wants it defined, as opposed to having two separate routes with and without the param.